usd_qtpy/editor.py

```diff
@@ -28,6 +28,36 @@ def __init__(self, stage, parent=None):
         splitter = QtWidgets.QSplitter(self)
         layout.addWidget(splitter)
 
+        # Menu bar
+        menubar = QtWidgets.QMenuBar()
+
+        file_action = menubar.addAction("File")
+        tools_action = menubar.addAction("Tools")
+        about_action = menubar.addAction("About")
+
+        def print_label(action : QtWidgets.QAction):
+            print(action.text())
+
+        menubar.triggered.connect(print_label)
+
+        file_menu = QtWidgets.QMenu()
+        tools_menu = QtWidgets.QMenu()
+        about_menu = QtWidgets.QMenu()
+
+        file_action.setMenu(file_menu)
+        tools_action.setMenu(tools_menu)
+        about_action.setMenu(about_menu)
```

Comment:
You should be able to create the menus directly like this
```suggestion
        file_menu = menubar.addMenu("File")
        tools_menu = menubar.addMenu("Tools")
        about_menu = menubar.addMenu("About")

        def print_label(action : QtWidgets.QAction):
            print(action.text())

        menubar.triggered.connect(print_label)
```